data/language/english_uk.txt

```diff
@@ -4111,6 +4111,19 @@ STR_5802    :{SMALLFONT}{BLACK}Stops guests from littering and vomiting
 STR_5803    :{SMALLFONT}{BLACK}Rotate selected map element
 STR_5804    :Mute sound
 
+#tooltips
+STR_5805    :{SMALLFONT}{BLACK}Use GPU for graphics instead of the CPU
+STR_5806    :{SMALLFONT}{BLACK}Disables FPS limit. Default limit is 40 FPSSet screen resolution
+STR_5807    :{SMALLFONT}{BLACK}Display FPS counter in-game
+STR_5808    :{SMALLFONT}{BLACK}Sets ratio to scale the game by.{NEWLINE}Most useful when playing in high resolutions
+STR_5809    :{SMALLFONT}{BLACK}[Requires hardware scaling]{NEWLINE}Sets UI scaling type
+STR_5810    :{SMALLFONT}{BLACK}[Requires hardware scaling]{NEWLINE}Use nearest neighbour scaling{NEWLINE}at integer scales
```

Comment:
I don't like that tip and I don't know what it should be, but the feature here allows the user to make sure the display is crisp when the scale factors of 2, 3, 4… useful to those who change this value often.